SUMMARY.md

```diff
@@ -24,6 +24,10 @@
 * [Tags and placeholders](applications/advanced-concepts/tags-and-placeholders.md)
 * [Multi-engine machine translation](applications/advanced-concepts/multi-engine-machine-translation.md)
 
+## Postediting
+
+* [HOW TO CONNECT POPULAR MTs TO CAT TOOLS](api_mt/index.md)
```

Comment:
Please note that other article names are spelled in lowercase.

Reply:
```
# Integrations
  * [CAT tools](/integrations/cat-tools.md)
```
And then in the article, the description should be *How to connect CAT tools to machine translation APIs*

Reply:
Ok. I changed it to: How to connect CAT tools to machine translation APIs. No italics. Will stick to the guidelines next time.